File: Evento/Correo.java
package Evento;
import gestionApp.Negocio;
import java.time.LocalDateTime;


public class Correo extends Evento{
    
    double cantidadPalabras;

    public Correo(int ID, Negocio negocio, String descripcion){
        super(ID, negocio, descripcion, LocalDateTime.now());
        this.cantidadPalabras = Math.random();

        this.darRespuesta();
    }

    private void darRespuesta(){
        
        if ((negocio.getEmpleadoEncargado().getHabilidadVenta() + negocio.cliente.getProbCompra() + this.cantidadPalabras)/3> 0.7){
            super.respuesta = "Interesado";
        }
        else{
            super.respuesta = "No interesado";
        }

    }

}

File: gestionApp/eventos/Correo.py
from gestionApp.eventos.Evento import Evento


class Correo(Evento):
    def __init__(self, negocio, descripcion, fecha, cantidadPalabras):
        super().__init__(negocio, descripcion, fecha)
        self.cantidadPalabras = cantidadPalabras
